data/keepRight.json

```diff
@@ -0,0 +1,393 @@
+{
+  "errorTypes": {
+    "20": {
+      "title": "multiple nodes on the same spot",
+      "severity": "warning",
+      "description": "There is more than one node in this spot. Offending node IDs: $1",
+      "IDs": ["20"],
+      "regex": "IDs: ((?:#\\d+,?)+)"
+    },
+    "30": {
+      "title": "non-closed_areas",
+      "severity": "error",
+      "description": "This way is tagged with '$1' and should be closed-loop.",
+      "regex": "'(.+)'"
+    },
+    "40": {
+      "title": "dead-ended one-ways",
+      "severity": "error",
+      "description": "The first node (id $1) of this one-way is not connected to any other way",
+      "IDs": ["n"],
+      "regex": "\\(id (\\d+)\\)"
+    },
+    "41": {
+      "title": "",
+      "severity": "error",
+      "description": "The last node (id $1) of this one-way is not connected to any other way",
+      "IDs": ["n"],
+      "regex": "\\(id (\\d+)\\)"
+    },
+    "42": {
+      "title": "",
+      "severity": "error",
+      "description": "This node cannot be reached because one-ways only lead away from here"
+    },
+    "43": {
+      "title": "",
+      "severity": "error",
+      "description": "You cannot escape from this node because one-ways only lead to here"
+    },
+    "50": {
+      "title": "almost-junctions",
+      "severity": "error",
+      "description": "This node is very close but not connected to way #$1",
+      "IDs": ["w"],
+      "regex": "way #(\\d+)"
+    },
+    "60": {
+      "title": "deprecated tags",
+      "severity": "warning",
+      "description": "This $1 uses deprecated tag $2. Please use $3 instead!",
+      "regex": "This (node|way|relation) uses deprecated tag '(.+)'\\. Please use &quot;(.+)&quot;"
+    },
+    "70": {
+      "title": "missing tags",
+      "severity": "error",
+      "description": ""
+    },
+    "71": {
+      "title": "",
+      "severity": "error",
+      "description": "This way has no tags"
+    },
+    "72": {
+      "title": "",
+      "severity": "error",
+      "description": "This node is not member of any way and does not have any tags"
+    },
+    "73": {
+      "title": "",
+      "severity": "error",
+      "description": "This way has a tracktype tag but no highway tag"
```

Comment:
Example of where this error requires regex:

https://www.keepright.at/report_map.php?schema=86&error=93785774

Reply:
Thanks good catch @SilentSpike !

from the source https://github.com/keepright/keepright/blob/dc7a8ada2c87f7cf68d9a1a546cebb53a1f0c281/checks/0070_missing_tags.php#L97-L108

Reply:
Oh nice, I hadn't even realised source was available for keepright 😄